appliances/Harbor/appliance.sh

```diff
@@ -210,6 +211,32 @@ download_unpack_harbor() {
 }
 
 
+mount_persistent() {
+    # Check if /data directory exists, if not create it
+    if [ ! -d "/data" ]; then
+        mkdir -p /data
+    fi
+
+    # Check if /dev/vda exists
+    if [ -e "/dev/vda" ]; then
+        # Check if already mounted
+        msg info "Persistent disk (/dev/vda) detected"
+        if grep -qs '/dev/vda' /proc/mounts; then
+            msg info "/dev/vda is already mounted."
+        else
+            # Add entry to /etc/fstab for persistence
+            echo "/dev/vda   /data   ext4   defaults   0   2" | sudo tee -a /etc/fstab > /dev/null
```

Comment:
why ext4? The volume can be xfs, we may get this from the VM template but maybe it is not there so we may need to use a blkinfo or similar